### ISSUE TITLE
1st movie/animation making using Blender

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#1127]
Started doing in https://github.com/kinu-garage/hut_10sqft/issues/945 that I have yet to finalize. Now I got a different project that requires moving making (related https://github.com/kinu-garage/hut_10sqft/issues/1086) so working on it using Blender again (kind of started logging in https://github.com/kinu-garage/hut_10sqft/issues/1122#issuecomment-2520645114).

Will keep posting my FI memos before completion.

**Comment by 130s** [comment]
Problem. I keep getting an error where a text strip disappears once a video starts. Asked on [blender.stackexchange.com#328298](https://blender.stackexchange.com/questions/328298/vse-text-strip-disappears-when-a-video-strip-starts).

**Comment by 130s** [comment]
# Blender operation cheat sheet 

After a few weeks since the last time I worked on Blender VSE I already forgot some shortcut keys frequently in use so that it matters in productivity.

Convention taken from ["Common Shortcuts" (blender.org)](https://docs.blender.org/manual/en/latest/interface/keymap/introduction.html)

Cheat sheet for Blender VSE:
- Making Meta Strip: Select all strips to be included in the MS, then [ctrl + G](https://youtu.be/MpDks9MKvXU?si=hKm2LtKNAXdDHN02&t=428) (from Ryan King Art).
- Looking into Meta Strip: Select a MS, then `tab`.
- Horizontally move cursor on the sequence: `shift` + `MMB`
- Split the selected Strip: TBD

**Comment by 130s** [comment]
Issue: Unsure how to make a part of strip fast-foward?

Tried: [Ryan Kings' speed control](https://www.youtube.com/watch?v=C80XtD2Q--w&list=PLsGl9GczcgBs0r5xcKL7KvrmGoRuro8RV&index=7). "Add -> Effect Strip -> Speed Control" fails with the msg "cannot apply effect audio sequence strip".

-> Which is resolved by "Disconnet"ing audio and image strips (right-click on the strip then the menu to do so will be there). But I need to understand this more in order to get the effect I want.

I ended up making a meta strip by combining audio + image strips then `Speed Control` can now be added to that meta strip.

But I still think I was able to fast-forward without making a meta strip..
